templates/components/products/product-view.html

```diff
@@ -248,9 +248,7 @@ <h2 class="productView-brand"{{#if schema}} itemprop="brand" itemscope itemtype=
                 {{> components/common/wishlist-dropdown}}
             {{/if}}
         </div>
-        {{#unless is_ajax}}
-            {{> components/common/share}}
-        {{/unless}}
+        {{> components/common/share url=product.url}}
```

Comment:
Passing in `url` explicitly, which makes this component more re-usable in other contexts if desired.